tango_simlib/examples/override_class.py

```diff
@@ -500,58 +499,30 @@ def test_action_stoprainstorm(self, model, tango_dev=None, data_input=None):
 
 class OverrideDish(object):
 
-    def _wait_while_configuring(self):
-        time.sleep(2)
-
-    def action_capture(self, model, tango_dev=None, data_input=None):
-        """Start/Stop capture on the configured band. Command only valide in SPFRx
-        Data_Capture mode.
-
-        data_input: str
-            ON/OFF value
-        """
-        raise NotImplementedError("The command handler for the command Capture"
-                                  " is not implemented.")
-
-    def action_configureattenuation(self, model, tango_dev=None, data_input=None):
-        """Set the global attenuation. Changing this value will set the attenuation
-        across the system and will be applied to all bands.
-
-        data_input: float
-            db TBC
-        """
-        raise NotImplementedError("The command handler for the command"
-                                  " ConfigureAttenuation is not implemented.")
-
-    def action_configureband1(self, model, tango_dev=None, data_input=None):
-        """This command triggers the Dish to transition to the CONFIGURE Dish Element
-        Mode, and returns to the caller. To configure the Dish to operate in frequency
-        band 1. On completion of the band configuration, Dish will automatically
-        revert to the previous Dish mode (OPERATE or STANDBY-FP).
-
-        data_input: str
-            timestamp
-        """
+    def _configureband(self, model, timestamp, band_number):
         _allowed_modes = ('STANDBY-FP', 'OPERATE')
         dish_mode_quant = model.sim_quantities['dishMode']
         current_mode_enum_val = dish_mode_quant.last_val
         current_mode_str_val = (
             dish_mode_quant.meta['enum_labels'][int(current_mode_enum_val)])
         if current_mode_str_val in _allowed_modes:
             set_mode = dish_mode_quant.meta['enum_labels'].index('CONFIG')
-            model.sim_quantities['dishMode'].set_val(set_mode, long(data_input[0]))
-            MODULE_LOGGER.info("Configuring DISH to operate in frequency band 1.")
-            self._wait_while_configuring()
-            MODULE_LOGGER.info("Done configuring DISH to operate in frequency band 1.")
+            model.sim_quantities['dishMode'].set_val(set_mode, long(timestamp))
```

Comment:
- [ ] Just double check this conversion to long here. Is it neccesary? Are we dealing with seconds or milliseconds?